packages/slice-machine/src/features/slices/convertLegacySlice/ConvertLegacySliceAsNewSliceDialog.tsx

```diff
@@ -36,6 +45,37 @@ export const ConvertLegacySliceAsNewSliceDialog: FC<DialogProps> = ({
     remoteSlices: getRemoteSlices(store),
   }));
 
+  const defaultValues = {
+    from: libraries[0]?.name,
+    sliceName: pascalize(slice.key),
+  };
+
+  const [values, setValues] = useState<FormValues>(defaultValues);
+
+  const [errors, setErrors] = useState<SliceModalValuesValidity>();
```

Comment:
If we keep an empty object here we don't have to check for errors all the time before checking if it has content.

Reply:
Hum, typescript don't like it, because of the type `SliceModalValuesValidity` that is shared with others form I didn't touch. It include the fact it can be undefined. WDYT?

Reply:
I did it for the other form, where form errors type is scoped